app/javascript/crayons/MentionAutocompleteTextArea/MentionAutocompleteTextArea.jsx

```diff
@@ -168,6 +166,12 @@ export const MentionAutocompleteTextArea = forwardRef(
       });
     }, [searchTerm, fetchSuggestions, cachedSearches, ariaHelperText]);
 
+    const { defaultValue = '' } = inputProps;
+
+    useEffect(() => {
```

Comment:
This is the bug fix. The `inputProps.onChange()` coming from the EditorBody component causes a rerender with the reset value. We were not updating this before. We were only setting the text content to the defaultValue when the `useState` hook for text content was being initialized.